UEBPrep/app/controllers/cards_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CardsController < ApplicationController
+  def new
+    @user = current_user
```

Comment:
Good call, stack overflow fails yet again with providing the optimal solution. The assumption totally makes sense, I just wasn't sure how to do the devise thing or if it was available yet and originally it was throwing an error when @user was nil (I wasn't logged in when checking it), but I might have moved that around so it doesn't matter now.